gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/features/StepDefinition.java

```diff
@@ -290,6 +298,15 @@ public void iteratedToList() {
         }
     }
 
+    @When("debug iterated to list")
```

Comment:
are the debug step definitions still relevant? or can they just be removed?

Reply:
if you keep them, could you please add documentation for them here: https://tinkerpop.apache.org/docs/current/dev/developer/#_when

Reply:
removed it. it's easy enough to add back in when needed